src/patternfly/components/DataList/data-list.scss

```diff
@@ -222,41 +224,26 @@ $pf-data-list--breakpoint--mobile: 380px;
   margin-left: var(--pf-c-data-list__action--MarginLeft);
 }
 
-// add keyframes to prevent flicker when expandable content scrolls
-@keyframes pf-c-data-list--expandable-content--in {
-  0% { overflow: hidden; }
-  99% { overflow-y: hidden; }
-  100% { overflow-y: auto; }
-}
-
 // Expandable content
 .pf-c-data-list__expandable-content {
+  position: relative;
   flex: 1 0 100%;
   align-self: flex-start;
   order: 2;
-  max-height: 0;
-
-  // reset li padding on expandable content
-  padding-right: var(--pf-c-data-list__expandable-content--PaddingRight);
-  padding-left: var(--pf-c-data-list__expandable-content--PaddingLeft);
+  max-height: var(--pf-c-data-list__expandable-content--MaxHeight);
 
   // offset for li padding for border to extend to li edges
   margin-right: var(--pf-c-data-list__expandable-content--MarginRight);
   margin-left: var(--pf-c-data-list__expandable-content--MarginLeft);
-  overflow: hidden;
+  overflow-y: auto;
   border-top: 0 solid var(--pf-c-data-list__expandable-content--BorderTopColor);
   box-shadow: var(--pf-c-data-list__expandable-content--BoxShadow);
-  opacity: 0;
-  transition: var(--pf-c-data-list__expandable-content--Transition);
 
   .pf-c-data-list__item.pf-m-expanded & {
-    max-height: var(--pf-c-data-list__item--m-expanded__expandable-content--MaxHeight);
-    padding-top: var(--pf-c-data-list__item--m-expanded__expandable-content--PaddingTop);
-    padding-bottom: var(--pf-c-data-list__item--m-expanded__expandable-content--PaddingBottom);
     border-top-width: var(--pf-c-data-list__item--m-expanded__expandable-content--BorderTopWidth);
-    opacity: 1;
+  }
 
-    // use animation to control overflow value to prevent flicker if content scrolls
-    animation: var(--pf-c-data-list__item--m-expanded__expandable-content--TransitionDuration) pf-c-data-list--expandable-content--in forwards;
+  .pf-c-data-list__expandable-content-body {
```

Comment:
Can you document both `.pf-c-data-list__expandable-content` and `.pf-c-data-list__expandable-content-body` in the expandable section code?